docs/UserGuide.md

```diff
@@ -100,13 +100,13 @@ StoreMando does not allow two identical items to exist.
 Two items are considered to be similar if they have the same spelling for name and location but differ by letter case. 
 StoreMando will accept similar items but will give a warning to alert the user about the existence of similar items.
 
-Item Attribute | Remarks
+Item Attribute | Constraints
 --------------------| -------
-Item Name  | An alphanumeric string that can contain spaces.
-Location | A string that can contain spaces and symbols.
-Quantity | An integer that must be at least 1.
+Item Name  | An alphanumeric string that can be one word or more.
+Location | A string.
+Quantity | An integer that must be at least 1 and cannot exceed 1,000,000.
 Expiry Date | A date in the format of YYYY-MM-DD. An item can either have 1 expiry date or no expiry date.
-Tag | An alphanumeric string that cannot contain spaces. An item can 0 or more tags.
+Tag | An alphanumeric string that can only be one word long. An item can 0 or more tags.
```

Comment:
Should this be "An item can have 0 or more tags."?

Reply:
yes i changed it. thx!